modules/ROOT/pages/tuning-pooling-profiles.adoc

```diff
@@ -0,0 +1,21 @@
+= Pooling Profiles
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Unlike singleton components, pooled components configure their component pool which contains multiple instances of the component to handle simultaneous incoming requests. Each pooled component has its pooling profile.
```

Comment:
This is also confusing: "Each pooled component has its pooling profile."

Does this really mean that each pool of components has a pooling profile,
or that each component in the pool has its own individual profile,
or that each component in the pool has the same profile as the rest of the components in that same pool?

Or does this mean that each pool of connections has a profile
or each connection in the pool has its own,
or each connection in the pool as the same as the rest?

I think the use of "component" in what is confusing. What is being pooled?

Reply:
```suggestion
Instead of using a single connection, improve request processing by Anypoint Connectors by pooling groups of individual connections to process simultaneous incoming requests. Each pool of connections has a unique pooling profile that defines the pool constituency.
```
Can we describe what this profile consists of and why it is necessary?